### PR TITLE
Build for node 7

### PR DESCRIPTION
**Pull request by hughrawlinson**

### CODE REVIEW

**Comment by hughrawlinson** [comment]
@nevosegal update: node 7 is actually out now. unfortunately, spectral flatness is off by  5.55111512e-17, so tests fail. Pretty unfortunate...

**Comment by hughrawlinson** [comment]
I am annoyed about this.

```
1) spectralFlatness should return correct Spectral Flatness value:
     AssertionError: expected 0.43959081704043357 to equal 0.4395908170404335
      at Context.<anonymous> (test/extractors/spectralFlatness.js:14:12)
```

**Comment by hughrawlinson** [comment]
ATTN: This PR should be squash-merged, not rebased or merge-committed.

**Comment by hughrawlinson** [comment]
@nevosegal @jakubfiala we can now support node 7! 😄

File: test/extractors/mfcc.js [hunk 1]
```diff
@@ -29,8 +29,11 @@ describe('mfcc', function () {
       -0.5572731218776171,
       -0.5245475651926765,
       -0.6215721667512493
-    ]
-    );
+    ];
+    
+    for (var index in expectedValues) {
+      assert.approximately(en[index], expectedValues[index], 0.000000000000001);
```
Comment (jakubfiala) [inline test/extractors/mfcc.js]: personally I'd use `1e-15` here and in the `spectralFlatness` test, it means people won't have to count the zeroes 😉 just a little nitpick that is

**Comment by hughrawlinson** [comment]
Totally agreed, I thought of using scientific notation when I was in bed
last night and it was a bad idea to take my computer out again. I had
forgotten that JS had scientific notation literals. I shall fix it shortly
:)

**Comment by hughrawlinson** [comment]
@jakubfiala - changed to sci notation literals. Just waiting for the build to pass. Used 1e-15 for mfcc because the aggregation increases the error margin. 1e-16 works for Spectral Flatness, so I went for the most precision possible in both cases 😄  Just waiting for tests to pass and we should be good for merge and release!

**Comment by hughrawlinson** [comment]
If we had a mailing list or a changelog we could announce that if people have stored features in a database and are using them for lookup, if they upgrade to node 7 that'll break their lookups. Do you think we should put something in the readme? I think it's highly unlikely that anyone has actually done that, and I would hope that if they did encounter a problem they'd file an issue.

**Pull request merged**